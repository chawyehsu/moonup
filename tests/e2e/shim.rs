use std::process::Command;

use insta_cmd::{assert_cmd_snapshot, get_cargo_bin};

const BIN: &str = "moonup-shim";

fn cli() -> Command {
    Command::new(get_cargo_bin(BIN))
}

#[test]
fn test_run_moonup_shim_directly() {
    assert_cmd_snapshot!(cli());
}
