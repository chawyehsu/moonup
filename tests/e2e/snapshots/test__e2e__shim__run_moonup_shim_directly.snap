---
source: tests/e2e/shim.rs
info:
  program: moonup-shim
  args: []
---
success: false
exit_code: 1
----- stdout -----

----- stderr -----
Cannot run moonup-shim directly
